packages/plugin_platform_interface/pubspec.yaml

```diff
@@ -6,12 +6,12 @@ description: Reusable base class for Flutter plugin platform interfaces.
 #
 # This package is used as a second level dependency for many plugins, a major version bump here
 # is guaranteed to lead the ecosystem to a version lock (the first plugin that upgrades to version
-# 2 of this package cannot be used with any other plugin that have not yet migrated).
+# 3 of this package cannot be used with any other plugin that have not yet migrated).
 #
 # Please consider carefully before bumping the major version of this package, ideally it should only
-# be done when absolutely necessary and after the ecosystem has already migrated to 1.X.Y version
-# that is forward compatible with 2.0.0 (ideally the ecosystem have migrated to depend on:
-# `plugin_platform_interface: >=1.X.Y <3.0.0`).
+# be done when absolutely necessary and after the ecosystem has already migrated to 2.X.Y version
+# that is forward compatible with 3.0.0 (ideally the ecosystem have migrated to depend on:
+# `plugin_platform_interface: >=2.X.Y <4.0.0`).
 version: 2.0.0-nullsafety
```

Comment:
@amirh I saw you added this comment. FYI

Reply:
I think the risk of someone using plugin_platform_interface without passing a token (which will be the case where we'd break) is extremely low. OTOH if we bump this we fragment all federated plugins into 2 incompatible sets which may end up in multiple plugins locking to their older versions.

My vote is to not make this a major bump.

Reply:
discussed offline. This package will bump y instead of x.